lib/configuration/nest-configuration.loader.ts

```diff
@@ -1,38 +1,31 @@
 import { Reader } from '../readers';
 import { Configuration } from './configuration';
 import { ConfigurationLoader } from './configuration.loader';
-import { defaultConfiguration } from './defaults';
+import { Schema } from 'types-joi';
```

Comment:
`joi` provides built-in TS definitions now. `types-joi` shouldn't be required

Reply:
You mean I should change this line into `import { Schema } from 'joi';`? Unfortunately, it will throw complie error after `npm run build`. From my opinion, maybe we should import `types-joi` instead of `joi` in ts file all the time. See [here](https://github.com/sideway/joi/issues/1557). What do think? Glad to hear your suggestion.

Reply:
The issue you linked is outdated. Joi provides its own type definitions now and we should stick to them. `types-joi` should be removed

Reply:
Done.